ecosystem/sep-0018.md

```diff
@@ -50,6 +50,27 @@ wallet.eth
 * **Term REGEXP:** `[_a-z][_a-z0-9]*`
 * **Key REGEXP:** `[_a-z][_a-z0-9]*(\.[_a-z][_a-z0-9]*)*`
 
+## Namespace Registry
+
+To guarantee interoperability and avoid naming conflicts, vendor-specific data
+must live under their reserved namespace:
+
+- `app`: Space dedicated to vendor-specific keys. Keys must be defined under
+  `app.{subdomain}`, with subdomain's dot(s) replaced by underscores. For
+  example, "example.org" defines its keys under `app.example_org`.
```

Comment:
Subdomain is a well-defined term meaning that URLs such as `app.example.org` may be used as well. Subdomain is the base unit in web security, for example regarding local storage, cookies, SSL certificates, permissions.... Hence, it makes sense to use the same unit here.

As an example, GitHub pages relies on these properties to offer a well secured `stellar.github.io` page that is fully separated from activity on other `*.github.io` pages.

Reply:
It is common to refer to a full domain as just a domain. `app.example.org` is a domain, and it is also a subdomain or `example.org`. `example.org` is a domain, and it is also a subdomain of `org`. These definitions come from [RFC1034 Section 3.1](https://tools.ietf.org/html/rfc1034#section-3.1). This [wikipedia page](https://en.wikipedia.org/wiki/Subdomain) makes this clear too when it says a subdomain is a domain.

If we specify `domain` we're saying the value could be `example.org` or `mysubdomain.example.org` or any other subdomain path like `my.subdomain.for.example.org` and we do not need to be more specific and require the value have to be a subdomain since that is just more specific definition of a domain.

Can we make that change above?

Reply:
Thanks for the explanation. Let's do that then.